### DISCUSSION TITLE
PDF/A-3 Support in 6.2

### DISCUSSION THREAD

**Discussion by Plerx2493** [empira/PDFsharp#210] [Q&A]
In a [recent announcement](https://github.com/empira/PDFsharp/issues/208) you stated that the next preview will be released this or next month, does this version already contain support for PDF/A-3? If this version does not contain this feature can you give a rough estimate when you think it will be supported?

**Comment by TH-Soft** [comment]
Nobody announced support for PDF/A-3 in PDFsharp. Too early for rough estimates.
Limited support for PDF/A is there.
